beancount/scripts/doctor_test.py

```diff
@@ -189,7 +188,7 @@ def test_context(self, filename):
         self.assertRegex(rv.stdout, "50.02")
 
     @test_utils.docfile
-    def test_context_multiple_files(self, filename):
+    def test_context_multiple_files(self, filename: str):
```

Comment:
Spurious change.

Reply:
Bah that's no big deal

Reply:
Sure. But it is @trim21 approach at dealing with review comments that is really annoying. Not even after pointing it out that there is an unaddressed comment I have been able to have they recognize it. I wonder why I should spend time to review the PR if my parts of my reviews are simply ignored.